### PR TITLE
Add limit setting to latest posts widget

### LINKED ISSUE #2862: Latest Posts Widget should have limit options, not be forced to use 5
### Describe the feature

The Latest Posts Widget has a limit of 5 hardcoded into the template so that only 5 posts can ever be shown. It would be great if I could choose from more options, such as 5, 10, 15. Forums with higher activity will need to show more.

### PR DESCRIPTION
**Pull request by partydragen**
Closes https://github.com/NamelessMC/Nameless/issues/2862

### CODE REVIEW

File: modules/Forum/widgets/admin/latest_posts.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ *  https://github.com/NamelessMC/Nameless/
+ *  NamelessMC version 2.1.0
+ *
+ *  License: MIT
+ *
+ *  Latest posts widget settings
+ */
+
+if (Input::exists()) {
+    $errors = [];
```
Comment (supercrafter100) [inline modules/Forum/widgets/admin/latest_posts.php:13]: You are never actually assigning the errors to smarty?

Comment (supercrafter100) [reply modules/Forum/widgets/admin/latest_posts.php:13]: same with the success message

Comment (partydragen) [reply modules/Forum/widgets/admin/latest_posts.php:13]: > You are never actually assigning the errors to smarty?

It being done by widgets page, this is not a actuall page but a sub file with add the data to the widgets settings page

Comment (supercrafter100) [reply modules/Forum/widgets/admin/latest_posts.php:13]: ah that explains

**Review by supercrafter100 — APPROVED** [review]

**Review by samerton — APPROVED** [review]

**Pull request merged**